genesyscloud/frontend/README.md

```diff
@@ -0,0 +1,133 @@
+# UI Module Application Server
+
+## Introduction
+
+The purpose of this guide is to set up an application server that can be used in conjunction with the proxy server in the same directory to render Agent Assist suggestions in GenesysCloud.
+
+Although it is possible to set up this integration deployment on any hosting platform, these instructions will use [Google's Cloud Run](https://cloud.google.com/run/) or [App Engine](https://cloud.google.com/appengine).
+
+## Configuration
```

Comment:
Either link to the offical documents or reorder all of the content to be similar to the official doc. This is all out of order and confusing.

Reply:
Done, linked to official doc